packages/web/next-env.d.ts

```diff
@@ -4,3 +4,5 @@ declare module '*.png';
 declare module '*.jpg';
 declare module '*.jpeg';
 declare module '*.svg';
+declare module '@honeybadger-io/webpack';
+declare module '@honeybadger-io/js';
```

Comment:
Why is this necessary?

Reply:
It seemed to fix an issue i was having when running my local build but I've removed them now and all is fine - I think the issue was my environment node version had changed from 16.13.1  to stable.